ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -414,7 +414,7 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                                 </React.Fragment>
                             )}
                             onSelect={val => {
-                                ctx.navigation.goto(`./${val}`);
+                                ctx.navigation.goto(`/workflows/${val}`);
```

Comment:
thank you for submitting this PR. I think, to support BASH_HREF, you need to use `uiUrl("workflows/"+val`?

Reply:
thanks @alexec  for pointing that out. I fixed in https://github.com/argoproj/argo/pull/3906/commits/5f97e8e58859d16b56246abb6125a3e55d01585d